src/display/font_loader.js

```diff
@@ -365,9 +365,41 @@ var FontFaceObject = (function FontFaceObjectClosure() {
 
     getPathGenerator: function FontLoader_getPathGenerator(objs, character) {
       if (!(character in this.compiledGlyphs)) {
-        var js = objs.get(this.loadedName + '_path_' + character);
-        /*jshint -W054 */
-        this.compiledGlyphs[character] = new Function('c', 'size', js);
+        var cmds = objs.get(this.loadedName + '_path_' + character);
+        // Try to compile cmds into JS for MAXIMUM SPEED
+        var js = '';
+        var cmd, args;
+        try {
+          for (var i in cmds) {
```

Comment:
`for (var i in cmds)` is slow use `for (var i = 0, ii = cmds.length; i < ii; i++)` instead (here and below)